.storybook/main.ts

```diff
@@ -59,31 +59,26 @@ const config = {
       );
     }
 
-    /**
-     * Until Storybook migrates its own internal @emotion usage from
-     * v10 to v11, this allows us to maintain compatibility
-     */
-    const emotion11CompatibleConfig = {
```

Comment:
6.4 added the `emotionAlias` helper so we do not need to manually set this anymore: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#emotion11-quasi-compatibility